HW_2/HW_2.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "HW_2"
+author: "Antonietta"
+date: "2/6/2021"
+output:   
+  github_document: default
+  html_document: default
+  keep_md: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(gapminder)
+library(tidyverse)
+library(palmerpenguins)
+library(gapminder)
+library(ggplot2)
+
+```
+
+
+# Exercise 1 : Basic dplyr
+
+## Task 1: filter () subset gapminder  data to 3 countries; France, Japan, and Italy and pipe %>% to select country and gdpPercap 
+
+```{r filter , echo=FALSE}
+gapminder %>%
+  select(country, year, gdpPercap) %>%
+  filter(country %in% c ("France" , "Japan", "Italy"), year>1969 & year <1980)
+
+```
+           
+## Task 2: Make new variable for the change life expectancy to show they have dropped 
+
+```{r mutate , echo=FALSE}
+gapminder %>%
+  mutate (droplifeExp = lag(lifeExp))
+
+``` 
+
+
+## Task 3: filter gapminder to show max GDP per cap by each country
+```{r max, echo=FALSE}
+gapminder %>%
+  select(country, gdpPercap) %>%
+  mutate(gdpPercap_max = gdpPercap) %>%
+  group_by(country) %>%
+  filter(gdpPercap_max == max(gdpPercap_max))
+```
+
+
+## Task 4: Produce scatterplot of Canada's life expectancy vs GDP
+
+```{r scatterplot , echo=FALSE}
+gapminder %>%
+  filter(country == "Canada") %>%
+  ggplot() +
+  aes(x =lifeExp, y = gdpPercap) +
+  geom_point(fill = "green" , color = "green", alpha = .5 , size = 2, shape = 23) +
+  scale_y_continuous(trans = 'sqrt')
```

Comment:
I think this is square rooting the values and not putting them on a log 10 scale.

Reply:
Oh, I thought square rooting would be another form of putting it on a log scale. Thank you, I'll fix that :)

Reply:
From what I googled it looked like using log10 and sqrt after trans = indicates what function you want it to do, but I could be incorrect.